test/test_python_package_change_task.py

```diff
@@ -49,20 +49,25 @@ def test__extract_base_dirs(self):
 
     def test__transfom(self):
         # Simple package transformations with arbitrary separator
-        self.assertEqual(self._TASK._transform('google.service', '.', []),
-                         'google.cloud.proto.service')
-        self.assertEqual(self._TASK._transform('google/other', '/', []),
-                         'google/cloud/proto/other')
+        self.assertEqual(self._TASK._transform('google.service.v1', '.', []),
+                         'google.service_v1.proto')
```

Comment:
Any reason you removed the `cloud` from the path?

Reply:
`_transform` no longer _adds_ cloud if it is not already there. I sanity checked with Pub/Sub, Spanner, and Vision and this seems to consistently do the right thing.